cmake/toolchains/mt.runtime.win32.toolchain.cmake

```diff
@@ -27,6 +27,11 @@ if(use_static_runtime)
         foreach(build_type "" "_DEBUG" "_MINSIZEREL" "_RELEASE" "_RELWITHDEBINFO")
             set(flag_var "CMAKE_${lang}_FLAGS${build_type}_INIT")
             string(REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
```

Comment:
Shall we remove this? The flag is empty here. @ilya-lavrenov

Reply:
are they empty even when you build from scratch ?
If yes, then let's drop